templates/deno/src/id.ts.twig

```diff
@@ -1,9 +1,31 @@
 export class ID {
+    // Generate an hex ID based on timestamp
+    // Recreated from https://www.php.net/manual/en/function.uniqid.php
+    static #hexTimestamp(): string {
+        const now = new Date();
+        const secondsSinceEpoch = Math.floor(now.getTime() / 1000);
+        const msecs = now.getTime() - secondsSinceEpoch * 1000;
+        const microsecondsPart = msecs * 1000; // Convert milliseconds to microseconds
```

Comment:
Why don't we go to this directly like you do in: `    final msecs = now.microsecondsSinceEpoch - secondsSinceEpoch * 1000000;`